ispdb/tvnet.lv

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<clientConfig version="1.1">
+  <emailProvider id="tvnet.lv">
```

Comment:
Does this domain have significant usage, so that it merits to be added here? We only add large ISPs.

Reply:
Thank You for the review.
I will make changes Monday accordingly!
Tvnet.lv active usage is in tens of thousands users but does not exceed 100,000 mark.
I can remove it if it is not enough.
I work for this ISP and am one of the developers there.
I will test accounts on all domains to check if I missed something.
Most domains work the same with these differences:

* mail.ee has IMAP/SMTP as mail.ee instead of mail.mail.ee, other domains have mail.domain.tld
* tvnet.lv users use mail.inbox.lv SMTP/IMAP directly instead of mail.tvnet.lv